recipes/opyplus/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "opyplus" %}
+{% set version = "1.0.0.dev4" %}
```

Comment:
This looks like it is prerelease. Since the main conda forge channel is for stable releases, you can either make this release official, or use the `prerelease` channel by adding it in `conda_build_config.yaml`.